### PR TITLE
[WIP][DO NOT MERGE] test codecov CI

### PR DESCRIPTION
**Pull request by stewart-yu**
DO NOT MERGE!!!
 test codecov CI
xref  issues #9257

### CODE REVIEW

File: pilot/pkg/serviceregistry/kube/register_test.go [hunk 1]
```diff
@@ -112,3 +112,108 @@ func TestAddLabelsAndAnnotations(t *testing.T) {
 		t.Errorf("Got unexpected %v for annotation a1=av1", o.Annotations["a1"])
 	}
 }
+
+//func TestRegisterEndpoint(t *testing.T) {
```
Comment (ZackButcher) [inline pilot/pkg/serviceregistry/kube/register_test.go]: We can go ahead and merge this if you remove commented out test and add in follow on PR (or we can wait with this PR till these tests can be un-commented).

**Comment by ZackButcher** [comment]
Ah actually it looks like `TestDeRegisterEndpoint` is in fact failing in CI.

**Comment by stewart-yu** [comment]
> Ah actually it looks like `TestDeRegisterEndpoint` is in fact failing in CI.

yes, this PR's change seems wrong
If you have chance, you can review https://github.com/istio/istio/pull/8752

**Pull request closed without merging**